match-lms-release/README.md

```diff
@@ -13,19 +13,17 @@ name: Release
 on:
   push:
     branches:
```

Comment:
If this is a sample people will copy, I figured it should reflect the best practice.